python/samples/04.ai.b.messageExtensions.AI-ME/env/.env.local

```diff
@@ -6,6 +6,8 @@ APP_NAME_SUFFIX=local
 
 # Generated during provision, you can also add your own variables.
 BOT_ID=
+SECRET_BOT_PASSWORD=
```

Comment:
Suggest moving the secrets to `.user` file, so everything in `.env.local` can be shared without security concern (though it's gitignored by default).

Reply:
The F5 flow was complaining if I didn't insert it in this file (rather than the `.local.user` file) - do you know where this is being mandated?